intelmq/bots/outputs/smtp_batch/output.py

```diff
@@ -238,8 +238,7 @@ def prepare_mails(self):
                         continue
                 else:
                     # visible both warning and print
-                    print(f"Warning: {mail_record} timeout, too big to read from redis", flush=True)
-                    self.logger.warning(f"Warning: {mail_record} timeout, too big to read from redis")
+                    self.logger.warning(f"Warning: %s timeout, too big to read from redis", mail_record)
```

Comment:
```suggestion
                    self.logger.warning("Warning: %s timeout, too big to read from redis", mail_record)
```
remove the f-string here as well